### PR TITLE
Add openvslam_ros

### PR DESCRIPTION
**Pull request by ymd-stella**
This PR intends to improve maintainability of the ROS wrapper.
It includes `#274` changes.

### CODE REVIEW

File: ros/src/openvslam/src/openvslam_ros.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+namespace openvslam_ros {
+class system {
+public:
+    system(const std::shared_ptr<openvslam::config>& cfg, const std::string& vocab_file_path, const std::string& mask_img_path);
+    openvslam::system SLAM_;
+    std::shared_ptr<openvslam::config> cfg_;
+    ros::NodeHandle nh_;
+    image_transport::ImageTransport it_;
+    decltype(std::chrono::steady_clock::now()) tp_0_;
```
Comment (lesurp) [inline ros/src/openvslam/src/openvslam_ros.h:24]: `std::chrono::steady_clock::time_point` might be cleaner here I think?

Comment (ymd-stella) [reply ros/src/openvslam/src/openvslam_ros.h]: Thanks for reviewing it.
I've fixed it in 9f25af0 .

**Comment by shinsumicco** [comment]
Sorry for the very late reply and thank you so much for providing the nice PR.

Before I review this PR, could you tell me features you intend to add?

**Comment by ymd-stella** [comment]
This PR contains the following:
- Refactoring ROS1 wrapper
- Makes the ROS1 wrapper available for stereo and RGBD

**Comment by shinsumicco** [comment]
Thank you for providing us the information.
Let us review the PR.

**Pull request closed without merging**